lib/active_model/serializer/include_tree.rb

```diff
@@ -0,0 +1,77 @@
+module ActiveModel
+  class Serializer
+    # Description
+
+    class IncludeTree
+      class << self
+        # Builds an IncludeTree from a comma separated list of dot separated paths (JSON API format).
+        # @example `'posts.author, posts.comments.upvotes, posts.comments.author'`
+        #
+        # @param [String] included
+        # @return [IncludeTree]
+        #
```

Comment:
Well, no, actually, I do, because it allows for all class methods (private and public) to be kept in the same place.